esp/esp/program/models/class_.py

```diff
@@ -1002,6 +1005,16 @@ def cancel(self, email_students=True, include_lottery_students=False, explanatio
                 else:
                     send_mail(ssi_email_title, msgtext, from_email, to_email)
 
+        if text_students and self.parent_program.hasModule('GroupTextModule') and GroupTextModule.is_configured():
+            if self.students(student_verbs).distinct().count() > 0:
+                msgtext = render_to_string('texts/class_cancellation.txt', context)
+                students_to_text = PersistentQueryFilter.create_from_Q(ESPUser, Q(id__in=[x.id for x in self.students(student_verbs)]))
+                send_log = GroupTextModule.sendMessages(students_to_text, msgtext)
+            else:
+                send_log = "No students to text"
+        else:
+            send_log = "GroupTextModule is not enabled or configured"
```

Comment:
Wait, does `send_log` go anywhere?  If you want, you can `logging.info` it, but you can also just not, if you're not going to display it back to the user, in which case you can just remove it!